benchmark/bench_de_bruijn.ml

```diff
@@ -249,14 +249,14 @@ module Simple4 : Lambda = struct
     | Var of int
 
   type term = S.t * term_
-  
+
```

Comment:
Changes to this file are unrelated to replacing earley by pacomp. I created an other MR that includes your changes plus some cleanup: https://github.com/rlepigre/ocaml-bindlib/pull/28. Please, review and merge. We can then rebase your change on top of that.

Reply:
You are right, my emacs is removing all trailing space as soon as I edit the file and this gives bad commit with too many changes. We could have a sanity check for that in the workflow, as well as long lines?

Reply:
Yeah, I think I have code for that somewhere, I'll see if I can add it when I have a moment.